lib/miq_automation_engine/service_models/miq_ae_service_generic_object.rb

```diff
@@ -0,0 +1,11 @@
+module MiqAeMethodService
+  class MiqAeServiceGenericObject < MiqAeServiceModelBase
+    def method_missing(method_name, *args)
+      @object.send(method_name, *args)
+    end
+
+    def respond_to_missing?(method_name, *args)
```

Comment:
@bzwei Is this method needed?  This logic will be handled by the generic_object at the active_record layer.  Also, `respond_to_missing?` takes a second argument `include_all` which is a boolean, not `*args`.